.github/workflows/rust.yml

```diff
@@ -3,28 +3,58 @@ name: Rust
 on: [push]
 
 jobs:
-  build:
+
+  codestyle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Rust
+        uses: hecrj/setup-rust-action@v1
+        with:
+          components: rustfmt
+          rust-version: nightly
```

Comment:
Does this work when rustfmt is not available in the nightly? https://rust-lang-nursery.github.io/rust-toolstate/

Having the CI fail in this situation, as well as having no rustfmt is highly undesirable.

Our travis config has: `The license_template_path setting we use to verify copyright headers is only available on the nightly rustfmt.` If that's still true, we should mention this here.